src/metabase/lib/util.cljc

```diff
@@ -237,11 +231,11 @@
       :native (native-query->pipeline query)
       :query  (mbql-query->pipeline query))))
 
-(mu/defn ^:private non-negative-stage-index :- [:int {:min 0}]
+(mu/defn absolute-stage-number :- [:int {:min 0}]
```

Comment:
This is much better! Taking the file system path analogy even further, `canonical-stage-number` could be even less ambiguous.

Reply:
Done